tox.ini

```diff
@@ -1,6 +1,21 @@
 [tox]
-envlist = py37
+envlist = py37, py38, py39, py310, py311
+isolated_build = true
+
+[gh-actions]
+python =
+    3.8: py38, 
+    3.9: py39, flake8
+    3.10: py310,
+    3.11: py311,
 
 [testenv]
 extras = dev
-commands = pytest {posargs}
```

Comment:
What about `{posargs}` ?

Reply:
I think it was the, removed, slow marker